src/reusecore/PageHeader/pageHeader.style.js

```diff
@@ -12,6 +12,9 @@ const PageHeaderWrapper = styled.div`
             margin: 3rem auto;
             padding: 0 1rem 0;
             z-index: 99;
+            h1 {
```

Comment:
Don't apply this change to the component code, as it will effect other places as well which I think we don't want.

Reply:
Yes, I overlooked that as well, but the website looked more like the proposed design to me. I'll fix this anyway. Thanks!

Reply:
If it looks fine, then we can keep it. No issues.